### PR TITLE
Example restructuring

### PR DESCRIPTION
**Pull request by luisaforozco**
In this PR I checked the examples:
* 01.00-Logistic.jl
* 02.00-GrayScott.jl
* 02.01-GrayScott.jl

I mainly modified linting issues and added some extra clarity in the text.
I checked that the notebooks would be running after my modifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,75 +2,82 @@
...
+### How can we train a CNODE?
+A priori fitting
 #### Example 02.01: Learning the Gray-Scott model
-* *Link 02.01* 
+* [Example 02.01](examples/02.01-GrayScott.jl)
   
 A posteriori fitting
 #### Example 02.02: Learning the Gray-Scott model
-* *Link 02.02* 
+* [Examples 02.02](examples/02.01-GrayScott_wMultDTO.jl)
```
Comment (SCiarella) [inline README.md:57]:
```suggestion
* [Examples 02.02](examples/02.02-GrayScott.jl)
```

File: examples/01.00-Logistic.jl [hunk 2]
```diff
@@ -98,15 +102,15 @@ ClipAdam = OptimiserChain(Adam(1.0f-1), ClipGrad(1));
 # Finally we can train the NODE
 result_neuralode = Optimization.solve(optprob,
     ClipAdam;
-    ## Commented out the line that uses a custom callback to track loss over time
-    ##callback = callback,
+    # Commented out the line that uses a custom callback to track loss over time
+    #callback = callback,
```
Comment (SCiarella) [inline examples/01.00-Logistic.jl:106]:
```suggestion
    ## Commented out the line that uses a custom callback to track loss over time
    ##callback = callback,
```

Comment (SCiarella) [reply examples/01.00-Logistic.jl:106]: If single #, then Literate.jl will split into different cells of the ipynb

File: examples/coupling_functions/functions_NODE.jl [hunk 3]
```diff
@@ -51,11 +51,11 @@ function create_f_CNODE(F_u, G_v, grid, NN_u=nothing, NN_v=nothing; is_closed=fa
...
             # For the nn I want u and v concatenated in the channel dimension
             uv -> let u = uv[1], v = uv[2]
-                cat(u,v,dims=3)
+                cat(u, v, dims=3)
             end,
             # Apply the right hand side of the CNODE 
-            SkipConnection(NN_closure, (f_NN, uv) -> let u = uv[:, :,1,:], v = uv[:,:,2, :]
-                (F_u(u, v, grid_for_force)+f_NN[1], G_v(u, v, grid_for_force)+f_NN[2])
+            SkipConnection(NN_closure, (f_NN, uv) -> let u = uv[:,:,1,:], v = uv[:,:,2,:]
+                (F_u(u, v, grid_for_force)+f_NN[1], G_v(u, v, grid_for_force) + f_NN[2])
```
Comment (SCiarella) [inline examples/coupling_functions/functions_NODE.jl:58]:
```suggestion
                (F_u(u, v, grid_for_force) + f_NN[1], G_v(u, v, grid_for_force) + f_NN[2])
```

**Review by SCiarella — APPROVED** [review]

**Comment by SCiarella** [comment]
@luisaforozco  I have added example 02.02

**Comment by luisaforozco** [comment]
Thanks for adding the example to this branch 😄 
I will look at it and make the changes before merging.

**Pull request merged**